packages/react-admin/src/reducer/admin/auth.js

```diff
@@ -0,0 +1,67 @@
+import {
+    USER_CHECK_SUCCESS,
+    USER_LOGIN_SUCCESS,
+} from '../../actions/authActions';
+
+const initialState = {
+    user: null,
+    permissions: {},
+};
+
+const getPermissionsKey = (route, resource, routeParams) => {
```

Comment:
Could you add js doc and an example ?

Reply:
Added jsDoc on the top level export (inside `reducer/index.js`)